tests/components/axis/test_init.py

```diff
@@ -57,20 +58,15 @@
     device = await setup_axis_integration(hass)
     assert hass.data[axis.DOMAIN]
 
-    assert await axis.async_unload_entry(hass, device.config_entry)
+    assert await hass.config_entries.async_unload(device.config_entry.entry_id)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4925854)